static/js/src/navigation.js

```diff
@@ -273,22 +273,24 @@ if (accountContainer && accountContainerSmall) {
       function setupAllContextualMenus(contextualMenuToggleSelector) {
         // Setup all menu toggles on the page.
         var userToggle = document.querySelector(contextualMenuToggleSelector);
-        setupContextualMenu(userToggle);
-
-        // Add handler for clicking outside the menu.
-        document.addEventListener("click", function (event) {
-          var contextualMenu = document.getElementById(
-            userToggle.getAttribute("aria-controls")
-          );
-          var clickOutside = !(
-            userToggle.contains(event.target) ||
-            contextualMenu.contains(event.target)
-          );
-
-          if (clickOutside) {
-            toggleMenu(userToggle, false);
-          }
-        });
+        if (userToggle) {
+          setupContextualMenu(userToggle);
```

Comment:
Most of the changes are whitespace due to the if statement wrapping this code. Add `?w=1` to the diff page to see the true changes.